db/schema.rb

```diff
@@ -232,30 +231,34 @@
     t.string   "promo_image_content_type"
     t.integer  "promo_image_file_size"
     t.datetime "promo_image_updated_at"
-    t.boolean  "visible",                  :default => true
+    t.boolean  "visible",                        :default => true
     t.string   "facebook"
     t.string   "instagram"
     t.string   "linkedin"
-    t.integer  "owner_id",                                     :null => false
-    t.string   "sells",                    :default => "none", :null => false
+    t.integer  "owner_id",                                           :null => false
+    t.string   "sells",                          :default => "none", :null => false
     t.string   "confirmation_token"
     t.datetime "confirmed_at"
     t.datetime "confirmation_sent_at"
     t.string   "unconfirmed_email"
     t.datetime "shop_trial_start_date"
-    t.boolean  "producer_profile_only",    :default => false
-    t.string   "permalink",                                    :null => false
-    t.boolean  "charges_sales_tax",        :default => false,  :null => false
+    t.boolean  "producer_profile_only",          :default => false
+    t.string   "permalink",                                          :null => false
+    t.boolean  "charges_sales_tax",              :default => false,  :null => false
     t.string   "email_address"
-    t.boolean  "require_login",            :default => false,  :null => false
-    t.boolean  "allow_guest_orders",       :default => true,   :null => false
+    t.boolean  "require_login",                  :default => false,  :null => false
+    t.boolean  "allow_guest_orders",             :default => true,   :null => false
     t.text     "invoice_text"
-    t.boolean  "display_invoice_logo",     :default => false
-    t.boolean  "allow_order_changes",      :default => false,  :null => false
+    t.boolean  "display_invoice_logo",           :default => false
+    t.boolean  "allow_order_changes",            :default => false,  :null => false
+    t.boolean  "require_phone_number",           :default => true
+    t.boolean  "require_bill_address",           :default => true
+    t.boolean  "hide_comment_box",               :default => false
```

Comment:
What is this? Sorry, it seems like my previous reviews haven't been thorough enough. I keep finding odd things. This commit needs some attention. These new fields seem to be completely unrelated. It look like Julius had a different database schema version when he did the migration. And it's difficult to spot, because the tab indent of other unaffected lines changed as well. I would suggest to re-write this commit. And if these fields are intentional and needed, then they should be in a separate commit.

Reply:
I hadn't spotted that either. Yes, this looks like it's related to a different PR. @ltrls ?

Reply:
@mkllnk I've amended the commit.

Reply:
oh my bad >< I didn't know that when running a migration ActiveRecord also infered schema.rb from the existing structure of the DB. I guess I need to rollback new migrations when I checkout branches, I thought I could keep new or modified columns without them leaking... sorry about that